libs/partners/deepseek/langchain_deepseek/chat_models.py

```
@@ -239,3 +245,37 @@ def _convert_chunk_to_generation_chunk(
                         reasoning_content
                     )
         return generation_chunk
+
+    def _stream(
+        self,
+        messages: List[BaseMessage],
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[CallbackManagerForLLMRun] = None,
+        **kwargs: Any,
+    ) -> Iterator[ChatGenerationChunk]:
+        try:
+            yield from super()._stream(
+                messages, stop=stop, run_manager=run_manager, **kwargs
+            )
+        except ValueError as e:
+            raise ValueError(
```

Should we catch ValueError, or JSONDecodeError?

In this commit we were catching JSONDecodeError: https://github.com/langchain-ai/langchain/pull/29758/commits/4bd23d6eaa2736dfc188bec102680c7e4c6edaa5

I think we should raise the same error type as is emitted now (e.g., JSONDecodeError, if that's what the client raises).

---

Perfect thank you very much. I was very perplexed here. I will update it